docs/guides/vqa_guide/estimators.rst

```diff
@@ -0,0 +1,40 @@
+.. _estimators_guide:
+
+==========
+Estimators
+==========
+
+Estimators abstract away the method which is used to estimate a quantity. ``orquestra-vqa``provides 2 main estimator tools ``EstimationMethod`` and ``EstimationTaskFactory``.
+
+``EstimationMethod``
+====================
+
+A ``QuantumBackend`` objects can be either a simulator or hardware. On a simulator, we have direct access to the wavefunction so we can simply extract the expectation value of an observable :math:`O` by calculating :math:`<\psi| O |\psi>`.
+
+In light of this ``orquestra`` proveds two separate ways of obtaining expectation values:
```

Comment:
Well, in reality it provides more than this, as there are some extra estimators in `orquestra-vqa`.

Reply:
Done!